portal_pages/forms.py

```diff
@@ -30,3 +98,27 @@ class Meta:
         fields = [
             'file'
         ]
+
+
+class CaseStudyForm(SpamProtectedForm, ModelForm):
+
+    required_css_class = 'required'
+
+    class Meta:
+        model = CaseStudyPage
+        fields = [
+            'title', 'summary', 'date',
+            'marketplace_entry'
+        ]
+
+
+class DocumentForm(ModelForm):
+
+    class Meta:
+        model = Document
+        labels = {
+            'file': 'Your Flow Document'
+        }
+        fields = [
+            'file'
+        ]
```

Comment:
If all we are expecting here is a json file they would should maybe call this something more specific than a "Document" form and actually validate that the file we receive is a json file...